SearchEngine/lambda_SearchEngine/include/Server_EchoServer.h

```diff
@@ -0,0 +1,145 @@
+#ifndef __ECHOSERVER_H__
+#define __ECHOSERVER_H__
+
+#include <sw/redis++/redis++.h>
+#include <unistd.h>
+
+#include <iostream>
+#include "Server_Thread.h"
+#include "CppJieba.h"
+#include "KeyRecommander.h"
+#include "Server_TcpConnection.h"
+#include "Server_TcpServer.h"
+#include "Server_ThreadPool.h"
+#include "WebPageSearcher.h"
+#include "CacheManager.h"
+#include "log4pp.h"
+
+using std::cout;
+using std::endl;
+using namespace sw::redis;
+
+
+class MyTask {
+public:
+    MyTask(const string &msg, const TcpConnectionPtr &con,
+           CnDictionary &cndict, EngDictionary &engdict, sw::redis::Redis &redis)
+        : _msg(msg), _con(con), _cndcit(cndict), _engdict(engdict), _redis(redis) {
+        }
+
+    void process() {
+        // 切出msg的前1位int,设为flag
+        auto & _currCache = CacheManager::GetInstance(nullptr)->getCache(atoi(name));
```

Comment:
do not name local variable with the same style as member variable even they point to the same memeory block.